crates/bdk/tests/wallet.rs

```diff
@@ -88,6 +87,62 @@ fn test_get_funded_wallet_balance() {
     assert_eq!(wallet.get_balance().confirmed, 50000);
 }
 
+#[test]
+fn test_get_funded_wallet_sent_and_received() {
+    let (wallet, _) = get_funded_wallet(get_test_wpkh());
+    assert_eq!(wallet.get_balance().confirmed, 50000);
+    let mut tx_amounts: Vec<(Txid, (u64, u64))> = wallet
+        .transactions()
+        .map(|ct| (ct.node.txid, wallet.sent_and_received(ct.node.tx)))
+        .collect();
+    tx_amounts.sort_by(|a1, a2| a1.0.cmp(&a2.0));
+
+    assert_eq!(tx_amounts.len(), 2);
+    assert_matches!(tx_amounts.get(0), Some((_, (76_000, 50_000))))
```

Comment:
How did we know what values we are expecting here?

Reply:
Thanks for catching this, I got sloppy here. Will fix it and add better docs too.